app/src/main/java/com/mikhailovskii/weatherandroid/ui/adapter/StickersDiffUtilCallback.kt

```diff
@@ -0,0 +1,22 @@
+package com.mikhailovskii.weatherandroid.ui.adapter
+
+import androidx.recyclerview.widget.DiffUtil
+import com.mikhailovskii.weatherandroid.data.entities.StickerPack
+
+class StickersDiffUtilCallback : DiffUtil.ItemCallback<StickerPack>() {
+
+    override fun areItemsTheSame(oldItem: StickerPack, newItem: StickerPack): Boolean = true
+
+    override fun areContentsTheSame(oldItem: StickerPack, newItem: StickerPack): Boolean =
+        if (oldItem.stickers?.isNotEmpty() == true && newItem.stickers?.isNotEmpty() == true) {
+            (oldItem.title == newItem.title
+                    && oldItem.stickers?.get(0) == newItem.stickers?.get(0)
+                    && oldItem.price == newItem.price)
+        } else if ((oldItem.stickers == null && newItem.stickers == null)
+            || (oldItem.stickers?.isEmpty() == true && newItem.stickers?.isEmpty() == true)
```

Comment:
@SergeiMikhailovskii Your statement is not fully correct

This one is correct. Compare and think why...
```kotlin
((oldItem.stickers == null || oldItem.stickers?.isEmpty())
            && (newItem.stickers == null || newItem.stickers?.isEmpty() == true)
```